theblues/plans.py

```diff
@@ -0,0 +1,50 @@
+from collections import namedtuple
+import datetime
+
+from theblues.errors import (
+    log,
+    ServerError,
+)
+from theblues.utils import (
+    ensure_trailing_slash,
+    make_request,
+)
+
+Plan = namedtuple('Plan', 'url plan created_on description price')
+
+
+class Plans(object):
+
+    def __init__(self, url):
+        """Initializer.
+
+        @param url The url to the Plan API.
```

Comment:
Should the URL include the revision? If so, please let's avoid it. The blues library should know what revision it supports, not users. Also, please document that.